doc/testing_tools/testing_floating_points.qbk

```diff
@@ -83,23 +83,29 @@ BOOST_AUTO_TEST_CASE(test, * utf::tolerance(0.02))
 }
 ``` 
 
+Finally, note that comparisons for tolerance are also applied to `operator<` with semantics 'less by more than some tolerance`,
+and other relational operators. Also, the tolerance-based comparisons are involved when a more complicated expression tree is
+processed within the assertion body:
 
+[bt_example tolerance_03..tolerance applied in more complex expressions]  
 
-[section:floating_points_comparison_theory Theory]
+[/############################################################################################]
 
-In most cases it is unreasonable to use an `operator==(...)` for a floating-point values equality check.
-The simple, absolute value comparison based, solution for a floating-point values `u`,
-`v` and a tolerance `epsilon`:
+[section:floating_points_comparison_theory Tolerance-based algorithms]
```

Comment:
I am mitigated here. I liked the "Theory" name better here, I would make another section "Practice" or  "Design in the __UTF__" or "Implementation design", containing the "Implementation part", but not contained in this section and visible in the TOC